src/tools/jetbrains-plugin.md

```diff
@@ -139,7 +144,7 @@ by removing a directory.
 See the JetBrains website for more information.
 
 * [IntelliJ IDEA](https://www.jetbrains.com/idea/)
-  * [Dart support](https://www.jetbrains.com/help/idea/dart-support.html)
+  * [Dart PhpStorm Help](https://www.jetbrains.com/help/phpstorm/dart.html)
```

Comment:
That's not the right link. This one looks right:

https://www.jetbrains.com/help/idea/dart-2.html